Remove native_functions.yaml dependency from PowKernel

peterbell10 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #64499
* #64173
* #64172
* __->__ #64171
* #64170

Differential Revision: [D30728583](https://our.internmc.facebook.com/intern/diff/D30728583)

peterbell10 committed:
Remove native_functions.yaml dependency from PowKernel

[ghstack-poisoned]

peterbell10 committed:
Update on "Remove native_functions.yaml dependency from PowKernel"




[ghstack-poisoned]

peterbell10 committed:
Update on "Remove native_functions.yaml dependency from PowKernel"




[ghstack-poisoned]

peterbell10 committed:
Update on "Remove native_functions.yaml dependency from PowKernel"




[ghstack-poisoned]

ezyang commented:
@ezyang has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30728583).

peterbell10 committed:
Update on "Remove native_functions.yaml dependency from PowKernel"


Differential Revision: [D30728583](https://our.internmc.facebook.com/intern/diff/D30728583)

[ghstack-poisoned]

peterbell10 committed:
Update on "Remove native_functions.yaml dependency from PowKernel"


Differential Revision: [D30728583](https://our.internmc.facebook.com/intern/diff/D30728583)

[ghstack-poisoned]

peterbell10 committed:
Update on "Remove native_functions.yaml dependency from PowKernel"


Differential Revision: [D30728583](https://our.internmc.facebook.com/intern/diff/D30728583)

[ghstack-poisoned]

peterbell10 committed:
Update on "Remove native_functions.yaml dependency from PowKernel"


Differential Revision: [D30728583](https://our.internmc.facebook.com/intern/diff/D30728583)

[ghstack-poisoned]

ezyang committed:
Update on "Remove native_functions.yaml dependency from PowKernel"


Differential Revision: [D30728583](https://our.internmc.facebook.com/intern/diff/D30728583)

[ghstack-poisoned]

ezyang commented:
@ezyang has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30728583).

peterbell10 committed:
Update on "Remove native_functions.yaml dependency from PowKernel"


Differential Revision: [D30728583](https://our.internmc.facebook.com/intern/diff/D30728583)

[ghstack-poisoned]

ezyang committed:
Update on "Remove native_functions.yaml dependency from PowKernel"


Differential Revision: [D30728583](https://our.internmc.facebook.com/intern/diff/D30728583)

[ghstack-poisoned]

ezyang commented:
@ezyang has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30728583).

Commit:
Update on "Remove native_functions.yaml dependency from PowKernel"


Differential Revision: [D30728583](https://our.internmc.facebook.com/intern/diff/D30728583)

[ghstack-poisoned]

dagitses commented:
@dagitses has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30728583).

Commit:
Update on "Remove native_functions.yaml dependency from PowKernel"


Differential Revision: [D30728583](https://our.internmc.facebook.com/intern/diff/D30728583)

[ghstack-poisoned]

dagitses commented:
@dagitses has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30728583).

Commit:
Update on "Remove native_functions.yaml dependency from PowKernel"


Differential Revision: [D30728583](https://our.internmc.facebook.com/intern/diff/D30728583)

[ghstack-poisoned]

dagitses commented:
@dagitses has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30728583).

Commit:
Update on "Remove native_functions.yaml dependency from PowKernel"


Differential Revision: [D30728583](https://our.internmc.facebook.com/intern/diff/D30728583)

[ghstack-poisoned]

Commit:
Update on "Remove native_functions.yaml dependency from PowKernel"


Differential Revision: [D30728583](https://our.internmc.facebook.com/intern/diff/D30728583)

[ghstack-poisoned]

dagitses commented:
@dagitses has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30728583).
